deploy/olm-catalog/smart-gateway-operator/metadata/properties.yaml

```diff
@@ -1,4 +1,4 @@
-    olm.properties: '[{"type": "olm.maxOpenShiftVersion", "value": "4.10"}]'
+    olm.properties: '[{"type": "olm.maxOpenShiftVersion", "value": "4.12"}]'
```

Comment:
So this is something I messed up when I copied out of the CSV in the other PR. This first line should not exist in the properties.yaml.  It was copied there for reference to know what to put in to the file, but I should have deleted it.